demo/player-ovp.html

```diff
@@ -18,16 +18,16 @@
         },
         targetId: 'player-placeholder',
         provider: {
-          partnerId: 242,
+          partnerId: 1091,
           env: {
-            serviceUrl: 'https://api.nvq2.ovp.kaltura.com/api_v3',
-            cdnUrl: 'https://api.nvq2.ovp.kaltura.com'
+            cdnUrl: 'https://qa-apache-php7.dev.kaltura.com/',
```

Comment:
why update this ? we dont use qa apache anyway

Reply:
The other one is not working, and this is still working